esrally/metrics.py

```diff
@@ -1325,6 +1328,8 @@ def to_result_dicts(self):
         if plugins:
             result_template["plugins"] = list(plugins)
 
+        if self.track_revision:
+            result_template["track-revision"] = self.track_revision
```

Comment:
We'll need to adjust tests in [metrics_test.py](https://github.com/elastic/rally/blob/9b9c7e03b79a11414740fa8f543b7a58b162149b/tests/metrics_test.py#L911-L1087) to check this too, right?